UiPathTeam.Extensions/UiPathTeam.Extensions.Activities/Properties/Resources.resx

```diff
@@ -445,44 +477,44 @@
     <value>Please provide values for both {0} and {1}.</value>
     <comment>error message; {0} &amp; {1} are complementary properties</comment>
   </data>
-  <data name="BuildDictionary_DisplayName" xml:space="preserve">
-    <value>Build Dictionary</value>
+  <data name="GetDictionaryValues_DisplayName" xml:space="preserve">
+    <value>GetDictionaryValues</value>
     <comment>activity name</comment>
   </data>
-  <data name="BuildDictionary_Description" xml:space="preserve">
-    <value>Activity that takes in an array of keys and values and outputs a dictionary.</value>
+  <data name="GetDictionaryValues_Description" xml:space="preserve">
+    <value>Get all values in the dictionary as list of objects</value>
     <comment>activity description</comment>
   </data>
-  <data name="BuildDictionary_In_keys_DisplayName" xml:space="preserve">
-    <value>in_keys</value>
+  <data name="GetDictionaryValues_Dictionary_DisplayName" xml:space="preserve">
+    <value>Dictionary</value>
     <comment>property name</comment>
   </data>
-  <data name="BuildDictionary_In_keys_Description" xml:space="preserve">
-    <value>Input for the array of keys. </value>
+  <data name="GetDictionaryValues_Dictionary_Description" xml:space="preserve">
+    <value>Dictionary</value>
     <comment>property description</comment>
   </data>
   <data name="Input_Category" xml:space="preserve">
     <value>Input</value>
     <comment>category</comment>
   </data>
-  <data name="BuildDictionary_In_values_DisplayName" xml:space="preserve">
-    <value>in_values</value>
+  <data name="GetDictionaryValues_ContinueOnError_DisplayName" xml:space="preserve">
+    <value>ContinueOnError</value>
     <comment>property name</comment>
   </data>
-  <data name="BuildDictionary_In_values_Description" xml:space="preserve">
-    <value>Input for the array of values.</value>
+  <data name="GetDictionaryValues_ContinueOnError_Description" xml:space="preserve">
+    <value>Continue On Error</value>
```

Comment:
please add a better description for the ContinueOnError variable since this is displayed within the tooltip in Studio